contracts/Vault.vy

```diff
@@ -1181,7 +1181,6 @@ def addStrategy(
     assert self.token.address == Strategy(strategy).want()
 
     # Check strategy parameters
-    assert self.debtRatio + debtRatio <= MAX_BPS
```

Comment:
Not sure why you'd want to delete this, as it should ensure that `self.debtRatio <= 100%`, which is a necessary invariant for this code to work correctly

Reply:
update code. It is OK
https://github.com/yearn/yearn-vaults/blob/f5a576b8c9ae07130df28abedadcad90e49f93f0/contracts/Vault.vy#L1478-L1480
vault_debtLimit: uint256 =  vault_totalAssets
    vault_totalDebt: uint256 = self.totalDebt
    strategy_debtLimit: uint256 = self.strategies[strategy].debtRatio * vault_totalAssets / self.debtRatio